models/addresses/ethereum/addresses_ethereum_dex.sql

```diff
@@ -41,6 +41,7 @@ FROM (VALUES
       ('0x7a250d5630b4cf539739df2c5dacb4c659f2488d', 'Uniswap', 'Router02'),
       ('0xe592427a0aece92de3edee1f18e0157c05861564', 'Uniswap', 'V3 Router'),
       ('0x68b3465833fb72a70ecdf485e0e4c7bd8665fc45', 'Uniswap', 'V3 Router 2'),
+      ('0xb45a2dda996c32e93b8c47098e90ed0e7ab18e39', 'Uniswap', 'TransitSwapRouterV4'),
```

Comment:
hmm @soispoke why does it include this change?

Reply:
no idea!

Reply:
should be good now